src/components/accounts/GenerateAccount/PasswordDialog/passwordDialog.js

```diff
@@ -58,7 +58,7 @@ class PasswordDialog extends React.Component {
                         <div style={{ width: '100%' }}>
                             <div className={ styles.passwordLabel }>Enter a strong password</div>
                             <div className={ styles.passwordSubLabel }>
-                                Password needs for confirm all wallet operations.
+                                Password needs for confirm all account operations.
```

Comment:
Missed one ;)

"This password will be required to confirm all account operations."

Or something else like that if you prefer 🖊 otherwise :)